services/ui-src/src/utils/testing/setupJest.tsx

```diff
@@ -57,6 +57,20 @@ export const mockStateUser: UserContextI = {
   loginWithIDM: () => {},
 };
 
+export const mockHelpDeskUser: UserContextI = {
+  user: {
+    userRole: UserRoles.HELP_DESK,
+    email: "helpdeskuser@test.com",
+    given_name: "Clippy",
+    family_name: "Helperson",
+    full_name: "Clippy Helperson",
```

Comment:
😆

Reply:
Just using the names that were already in the file!! Debated making them all Futurama themed in the setupJest file.